src/OAuth/AccessKey.cs

```diff
@@ -20,5 +22,12 @@ public class AccessKey
 
         [JsonProperty("jwk")]
         public JsonWebKey Jwk { set; get; }
+
+        public static AccessKey CreateFromBase64EncodedAccessKey(string encoded)
```

Comment:
the parameter name `encoded` should be more descriptive of what this is. Rename to `base64EncodedAccessKey` i guess would be fine.